test/src/Game/Werewolf/Test/Game.hs

```diff
@@ -22,11 +22,8 @@ import Game.Werewolf.Player
 
 import Test.QuickCheck
 
-prop_newGameStartsWithSeersTurnWhenSeersPresent :: [Player] -> Property
-prop_newGameStartsWithSeersTurnWhenSeersPresent players = any isSeer players ==> isSeersTurn (newGame players)
-
-prop_newGameStartsWithWerewolvesTurnWhenSeersAbsent :: [Player] -> Bool
-prop_newGameStartsWithWerewolvesTurnWhenSeersAbsent players = isWerewolvesTurn (newGame $ filter (not . isSeer) players)
+prop_newGameStartsWithNightfallTurn :: [Player] -> Property
+prop_newGameStartsWithNightfallTurn players = any isSeer players ==> isNightfallTurn (newGame players)
```

Comment:
Sorry, last thing -> can just remove the `any isSeer players` from here and then it's good to be merged :).